wsmaster/che-core-api-metrics/pom.xml

```diff
@@ -39,6 +39,10 @@
             <groupId>org.eclipse.che.core</groupId>
             <artifactId>che-core-api-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.che.core</groupId>
+            <artifactId>che-core-api-dto</artifactId>
```

Comment:
I guess it should have `test` scope. To check it - please execute `mvn clean install -DskipTests` and maven dependency plugin will test the right scope.

Reply:
Good point, it should be in the test scope! maven dependency plugin doesn't complain though in either case...